src/main/java/com/graphbuilder/geom/PointFactory.java

```diff
@@ -41,29 +41,24 @@ public Point2D(double x, double y) {
 			pts = new double[]{x, y};
 		}
 		
-		@Override
 		public double getX() {
 			return pts[0];
 		}
 
-		@Override
 		public double getY() {
 			return pts[1];
 		}
 
-		@Override
 		public void setLocation(double[] p) {
 			pts[0] = p[0];
 			pts[1] = p[1];
 		}
 		
-		@Override
 		public void setLocation(double x, double y) {
 			pts[0] = x;
 			pts[1] = y;
 		}
 
-		@Override
```

Comment:
In Java 1.5 code compliance it was not usual to put override on interface implementation methods and eclipse per default marks them as errors, so i removed them. The alternative would have been to change it to ignore in eclipse or to push the code compliance level to 1.6 in the maven pom.xml. Though you are free to decide otherwise here.